Stack/Opc.Ua.Core/Security/Certificates/CertificateFactory.cs

```diff
@@ -1081,6 +1089,10 @@ private static RsaPrivateCrtKeyParameters GetPrivateKeyParameter(X509Certificate
                 new BigInteger(1, rsaParams.InverseQ));
             return keyParams;
         }
+        finally
+        {
+            RsaUtils.RSADispose(rsa);
```

Comment:
I think comments would help in these finally blocks.  If someone comes across this s/he might change it back to using() for good measure.  Best s/he understands that this is needed due to Mono not doing dispose() right.

Reply:
yep indeed

---
In reply to: [163232612](https://github.com/OPCFoundation/UA-.NETStandard/pull/325#discussion_r163232612) [](ancestors = 163232612)

Reply:
actually hovering over RSADispose provides a lengthy description of the issue...

---
In reply to: [163300493](https://github.com/OPCFoundation/UA-.NETStandard/pull/325#discussion_r163300493) [](ancestors = 163300493,163232612)